docs/source/item-validation.rst

```diff
@@ -193,6 +221,36 @@ as a `dict`:
         OtherItem: '/path/to/otheritem_schema.json',
     }
 
+.. _SPIDERMON_VALIDATION_CERBERUS:
+
+SPIDERMON_VALIDATION_CERBERUS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Default: ``None``
+
+A `list` containing the local path of the item schema.
+
+.. code-block:: python
+
+    # settings.py
+
+    SPIDERMON_VALIDATION_CERBERUS = [
+        '/path/to/schema.json',
+        'http://example.com/mycerberusschema',
+        {"Field": {"type": "number", "required":True}}
+    ]
+
+If you are working on a spider that produces multiple items types, you can define paths to schema for each item as `dict` as shown below:
+
+    # settings.py
```

Comment:
Please, use syntax highlighting here as well.

Reply:
Yes, just added this bit.